alien-bind.js

```diff
@@ -0,0 +1,17 @@
+module.exports = alienBind
+
+function alienBind (fn, context) {
+  var boundArgs = rest(arguments, 2)
+  return function () {
+    var args = boundArgs.concat(toArray(arguments))
+    return fn.apply(context, args)
+  }
+}
+
+function rest (list, index) {
+  return Array.prototype.slice.call(list, 2)
```

Comment:
`Array.prototype.bind` is overwritten in Mootools... but I see what you are getting at.